app/components/Reactions/ReactionPickerCategory.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react';
+import classNames from 'classnames';
+import styles from './ReactionPickerCategory.css';
+import Emoji from 'app/components/Emoji';
+
+type Props = {
+  isActive: boolean,
+  name: string,
+  onCategoryClick: (category: string) => void
+};
+
+const mapCategoryNameToIcon = name => {
+  switch (name) {
+    case 'people':
+      return '😃';
+    case 'animals_and_nature':
+      return '🍃';
+    case 'food_and_drink':
+      return '🍔';
+    case 'activity':
+      return '⚽';
+    case 'travel_and_places':
+      return '✈';
+    case 'objects':
+      return '💡';
+    case 'symbols':
+      return '❤';
+    case 'flags':
+      return '🇵🇱 ';
+    default:
+      return name.charAt(0).toUpperCase();
```

Comment:
What do you think about using the categories to set the icons? f.ex. the first emoji in the category, or maybe modifying the list of emojis in backend to set a  __main__ emoji in a category there?

Reply:
This is a good idea, but this will be a #TODO for a future PR, where we add a new model for emojis (i.e. `EmojiCategory`) where we can have a `emoji` field.